history.py

```diff
@@ -0,0 +1,31 @@
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+        # Code to be executed for each request before
+        # the view (and later middleware) are called.
+        product_ids = request.session.get('LastFiveList', [])
```

Comment:
I think it is best practice to use like lastFiveList instead of all caps

Reply:
I agree! Typically I always have my variables lower case, or camel case, but for some reason I like to have some of my methods or classes with upper case. However, I don't really follow my rule the whole time throughout my code, so I should change it